test/README.md

```diff
@@ -102,24 +102,20 @@ go test -v -tags=e2e -count=1 ./test/e2e --resolvabledomain
 The [`upload-test-images.sh`](./upload-test-images.sh) script can be used to build and push the
 test images used by the conformance and e2e tests. It requires:
 
-* [`DOCKER_REPO_OVERRIDE`](/DEVELOPMENT.md#environment-setup) to be set
+* [`KO_DOCKER_REPO`](/DEVELOPMENT.md#environment-setup) to be set
 * You to be [authenticated with your
   `DOCKER_REPO_OVERRIDE`](/docs/setting-up-a-docker-registry.md)
 * [`docker`](https://docs.docker.com/install/) to be installed
 
 To run the script for all end to end test images:
 
 ```bash
-./test/upload-test-images.sh ./test/e2e/test_images ./test/conformance/test_images
+./test/upload-test-images.sh test/test_images
```

Comment:
The location of the test images is now fixed, you don't need arguments for this script anymore (see my comment below).

Reply:
Done